routes/credentialsRoutes.js

```diff
@@ -0,0 +1,91 @@
+const router = require("express").Router();
+const { ContentCreator } = require("../models/ContentCreatorApplication");
+const bcrypt = require('bcrypt');
+const errorCodes = require('../helpers/errorCodes');
+
+const jwt = require("jsonwebtoken");
+const { compare, encrypt } = require("../helpers/password");
+const { signAccessToken } = require("../helpers/token");
+const user = require("../users/user");
+const { makeExpressCallback } = require("../helpers/express");
+const { authEndpointHandler } = require("../auth");
+
+router.post('/', makeExpressCallback(authEndpointHandler))
+/**
+* Signup Route.
+* Creates a new Content Creator Application.
+*
+* @param {JSON} Form Which includes the following fields:
+* @param {String} name Name of the Content Creator
+* @param {String} email Email of the Content Creator (Will be used for login)
+* @param {String} password Password of the Content Creator (Will be encrypted)
+* @returns {JSON} Returns response status code
+*/
+router.post("/signup", async (req, res) => {
+  const form = req.body;
+  
+  const email = req.body.email;
+  const hashpassword = encrypt(req.body.password)
```

Comment:
I might be wrong, but does this not mean that i hashes the password on the server/backend after it have received it from the frontend? This would mean that the password is send in its raw form, which is not optimal. I think the pull should still be accepted, but it should be changed in the future to fix this.

Reply:
For my money the password should be hashed in the backend, but sent over an https connection. Replay attacks and the like are still possible if you send a hashed password from the client.